source/Octopus.Tentacle.Tests.Integration/Util/TcpTentacleHelpers/ResponseMessageTcpKillerWorkarounds.cs

```diff
@@ -17,13 +17,6 @@ public static void EnsureTentacleIsConnectedToServer(this IClientScriptServiceV2
             logger.ForContext(typeof(ResponseMessageTcpKillerWorkarounds)).Information("Finished GetStatus work around call");
         }
 
-        public static void EnsureTentacleIsConnectedToServer(this IClientCapabilitiesServiceV2 service, ILogger logger)
```

Comment:
This can no longer be used to ensure tentacle is connected as responses are cached